src/components/common/view/AddWebApp.web.js

```diff
@@ -199,29 +205,19 @@ const AddWebApp = props => {
   }, [dialogShown])
 
   useEffect(() => {
-    log.debug({ installPrompt, show, lastCheck })
+    log.debug({ installPrompt, show, skipCount })
 
     // Condition to show reminder
     if (lastCheck) {
-      const DAYS_TO_WAIT = 5
-      const thirtyDaysFromLastDate = new Date()
-      thirtyDaysFromLastDate.setDate(lastCheck.getDate() + DAYS_TO_WAIT)
-      const today = new Date()
-      log.debug({
-        installPrompt,
-        show,
-        lastCheck,
-        today,
-        thirtyDaysFromLastDate,
-        DAYS_TO_WAIT,
-      })
-
-      if (thirtyDaysFromLastDate < today || lastCheck > lastClaim) {
+      const isNextCheckBanned = !!nextCheck && moment(nextCheck).isAfter(moment())
+      const notClaimedAfterLastCheck = lastClaim ? moment(lastCheck).isAfter(moment(lastClaim)) : true
+
+      if (isNextCheckBanned || notClaimedAfterLastCheck) {
         return
       }
     }
 
-    if ((installPrompt && show) || (isMobileSafari && show)) {
+    if (show || (isMobileSafari && show)) {
```

Comment:
strange condition. may be it should be 
`if(isMobileSafari && show)`
?

Reply:
@AlexeyKosinski yes, you are right. I changed it for test purposes. Reverted in a new commit.